packages/common/src/design-language/theme.ts

```diff
@@ -12,6 +12,9 @@ const theme = {
   // transition speeds in ms
   speeds: [0, '75ms', '100ms', '150ms', '200ms', '300ms', '500ms'],
 
+  // mobile first
+  breakpoints: ['576px', '768px', '992px'],
```

Comment:
Note: this was missing from our design language, added it for responsive components

Reply:
niiice! Danke!